packages/pangea-sdk/pom.xml

```diff
@@ -106,6 +106,11 @@
       <artifactId>httpmime</artifactId>
       <version>4.5.14</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpmime</artifactId>
```

Comment:
This dependency already exists on line 106.

Reply:
Removed